### DISCUSSION TITLE
全部关闭功能按钮标题错误

### DISCUSSION THREAD

**Discussion by mashuai720** [dbeaver/dbeaver#16825] [DBeaver Development]
![错误标题截图](https://user-images.githubusercontent.com/36294826/174304014-4a4519a4-40e5-4cfe-a1a8-39777aafa2cf.png)

**Comment by LonwoLonwo** [comment]
Please use English language.

**Comment by zhuwenbing** [comment]
Incorrect Chinese description of "Close all tabs" which has not been fixed.
I met the same problem.

**Reply by LonwoLonwo** [reply]
Hello @zhuwenbing 
Can you provide the correct one, please?

**Reply by zhuwenbing** [reply]
I can't find the corresponding i18n description in the DBeaver project. Maybe it's provided by eclipse?
The following may be a better description of this feature:
```
Chinese -> 关闭全部
ASCII -> \u5173\u95ed\u5168\u90e8
```

**Reply by LonwoLonwo** [reply]
Yeah, probably you are right about Eclipse
